app/Http/Controllers/Api/NotesController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Events\NoteAdded;
+use App\Helpers\Helper;
+use App\Http\Controllers\Controller;
+use App\Models\Asset;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class NotesController extends Controller
+{
+    public function store(Request $request)
+    {
+        $item = Asset::findOrFail($request->input("id"));
```

Comment:
Since this PR is focusing on adding notes to assets this is fine to be hardcoded but adding a note here would be helpful since the modal is passing along `type=asset` already. Something like 

```
// This can be made dynamic by using $request->input('type') to determine which model type to add the note to.
```